go/ai-proxy/args/args.go

```diff
@@ -76,27 +76,26 @@ func ProviderHost() string {
 	return *argProviderHost
 }
 
-func ProviderCredentials() []string {
-	if len(*argProviderTokens) > 0 && Provider() == api.ProviderOpenAI {
-		return *argProviderTokens
-	}
-
+func ProviderServiceAccount() string {
 	if len(*argProviderServiceAccount) > 0 && Provider() == api.ProviderVertex {
-		return []string{*argProviderServiceAccount}
+		return *argProviderServiceAccount
 	}
+	return ""
+}
 
-	if Provider() == api.ProviderBedrock {
-		if len(*argsProviderAWSRegion) > 0 {
-			return []string{*argsProviderAWSRegion}
-		}
-		return []string{}
+func ProviderAwsRegion() string {
+	if Provider() == api.ProviderBedrock && len(*argsProviderAWSRegion) > 0 {
```

Comment:
same this is almost certainly a nil pointer risk

Reply:
We should still fix the codepaths, it's entirely possible a future developer doesn't know to guard it upstream of that callsite.